kafka/pykafka/balancedconsumer.py

```diff
@@ -0,0 +1,371 @@
+import itertools
```

Comment:
Noting before reading the file: can this go one level up? It'd be ideal if the same BalancedConsumer could handle the Python and C implementations.